src/SbManager/Scripts/manager/statusController.js

```diff
@@ -0,0 +1,20 @@
+﻿$app.controller('statusController', ['$scope', function ($scope) {
+    $scope.refresh = function () {
+        $scope.model = null;
+        $.ajax({
+            url: window.applicationBasePath + "/api/v1/busmanager/",
+            dataType: 'json',
+            success: function (d) {
```

Comment:
I have addressed this in the newest commit, I also updated the home controller to remove the deprecated callbacks.